internal/pkg/cli/svc_deploy.go

```diff
@@ -200,9 +202,25 @@ func (o *deploySvcOpts) Execute() error {
 			RootUserARN: o.rootUserARN,
 			Tags:        tags.Merge(targetApp.Tags, o.resourceTags),
 		},
-		ForceNewUpdate: o.forceNewUpdate,
+		DeployOptions: deploy.DeployOptions{
+			ForceNewUpdate:  o.forceNewUpdate,
+			DisableRollback: o.disableRollback,
+		},
 	})
 	if err != nil {
+		if o.disableRollback {
+			stackName := stack.NameForService(o.targetApp.Name, o.targetEnv.Name, o.name)
+			svcLogsCmd := "copilot svc logs"
+			awsCLIRollbackCmd := fmt.Sprintf("aws cloudformation rollback-stack --stack-name %s --role-arn %s", stackName, o.targetEnv.ExecutionRoleARN)
+			svcDeployCmd := "copilot svc deploy"
+			log.Infof(`It seems like you have disabled automatic stack rollback for this deployment. To debug, you can:
+* Run %s to inspect the service log.
+* Visit AWS console to inspect the errors.
+After fixing the deployment, you can:
+1. Run %s to rollback the deployment.
+2. Run %s to make a new deployment.
+`, color.HighlightCode(svcLogsCmd), color.HighlightCode(awsCLIRollbackCmd), color.HighlightCode(svcDeployCmd))
```

Comment:
super nit: I think it's slightly less daunting if we remove some of these variables
```suggestion
`, color.HighlightCode("copilot svc logs"), color.HighlightCode(rollbackCmd), color.HighlightCode("copilot svc deploy"))
```